CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [7.1.0](https://github.com/theforeman/puppet-qpid/tree/7.1.0) (2021-03-29)
+
+[Full Changelog](https://github.com/theforeman/puppet-qpid/compare/7.0.0...7.1.0)
+
+**Implemented enhancements:**
+
+- Refs [\#32037](https://projects.theforeman.org/issues/32037): Add parameters to disable qpid services [\#148](https://github.com/theforeman/puppet-qpid/pull/148) ([ehelms](https://github.com/ehelms))
+
+**Fixed bugs:**
+
+- Remove duplicate relations [\#150](https://github.com/theforeman/puppet-qpid/pull/150) ([ekohl](https://github.com/ekohl))
```

Comment:
I debated about whether this should be labeled as a bug or merged PR.

Thinking about it more, maybe the best approach is to skip the changelog for this one entirely, since I don't think it should be relevant for an end user.